ckanext/who_afro/assets/css/who-afro.css

```diff
@@ -1008,3 +1425,45 @@ a.tag:hover  {
     margin-top: 20px;
     padding: 10px 0px;
 }
+
+.description {
+    color: var(--who-blue-ten-percent);
```

Comment:
This var() syntax has a compatability constraint that I think we are ok with (I think post 2016 browsers and we are good).  But it would probably be a good idea to think through and agree which versions of browsers we aim to support.